### PR TITLE
add types of zipkin-context-cls, zipkin-instrumentation-express, zipk…

### PR DESCRIPTION
**Pull request by plantain-00**
…in-instrumentation-fetch, zipkin-transport-http

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by plantain-00** [comment]
waiting https://github.com/Microsoft/types-publisher/pull/428 to be merged

File: types/zipkin-instrumentation-express/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+// Type definitions for zipkin-instrumentation-express 0.11
+// Project: https://github.com/openzipkin/zipkin-js#readme
+// Definitions by: York Yao <https://github.com/plantain-00>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
```
Comment (andy-ms) [inline types/zipkin-instrumentation-express/index.d.ts]: Add a comment `// TypeScript Version: 2.2` here (line 5) because `express` requires it.

File: types/zipkin-context-cls/zipkin-context-cls-tests.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    jsonEncoder,
+} from 'zipkin';
+
+import CLSContext = require('zipkin-context-cls');
+import { HttpLogger } from 'zipkin-transport-http';
+
+// Setup the tracer to use http and implicit trace context
+const tracer = new Tracer({
+    ctxImpl: new CLSContext('zipkin'),
+    recorder: new BatchRecorder({
```
Comment (andy-ms) [inline types/zipkin-context-cls/zipkin-context-cls-tests.ts]: This part in `recorder:` doesn't look relevant to testing `zipkin-context-cls` -- all we do to test that is construct it and pass it as `ctxImpl`.

Comment (plantain-00) [reply types/zipkin-context-cls/zipkin-context-cls-tests.ts]: done.
